package.json

```diff
@@ -10,7 +10,10 @@
     "@typescript-eslint/parser": "2.6.0",
     "bootstrap": "3.4.1",
     "bootstrap-colorpicker": "2.5.2",
-    "chart.js": "2.9.2",
+    "chart.js": "2.8.0",
+    "common-tags": "1.8.0",
```

Comment:
Could you remove `common-tags` and `@types/common-tags` now that we're no longer using `oneLineTrim`?